libGitWrap/Commit.cpp

```diff
@@ -347,23 +345,23 @@ namespace Git
 
     DiffList Commit::diffFromParent(Result& result, unsigned int index)
```

Comment:
It is renamed now to be convenient and consistent with lg2 api. After all it's just a preposition, that can be interpreted in both directions :).